src/Fixer/Basic/BracesFixer.php

```diff
@@ -846,6 +846,14 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
 
         $nextToken = $tokens[$nextTokenIndex];
         if ($nextToken->isComment()) {
+            $previousToken = $tokens[$nextTokenIndex - 1];
+            // do not indent inline comments used to comment out unused code
+            if (
+                0 === strpos($nextToken->getContent(), '//'.$this->whitespacesConfig->getIndent())
+                && $previousToken->isWhitespace() && 1 === preg_match('/\R$/', $previousToken->getContent())
```

Comment:
For me, while using PHP Storm, commented code looks exactly like this:
```diff
 if (1) {
-    // echo 2;
+    // echo 2;
 }
```

if I get you right, you also check that there is no single-line spaces before `//`, right ?